bin/start

```diff
@@ -2,11 +2,15 @@
 
 main(){
   case $1 in
-    -i) #start normally
+    -i) #start normally - used for jenkins
       docker-compose up -d
+      docker exec $(docker-compose ps -q gridium) bundle install;
```

Comment:
is the bundle install necessary? There is already one executed in the `Dockerfile`